persistence/application.go

```diff
@@ -25,11 +25,11 @@ func (p PostgresContext) GetApp(address []byte, height int64) (operator, publicK
 	return
 }
 
-func (p PostgresContext) InsertApp(address []byte, publicKey []byte, output []byte, _ bool, _ int, maxRelays string, stakedAmount string, chains []string, pausedHeight int64, unstakingHeight int64) error {
```

Comment:
Why did you reintroduce `paused` and `status` names in the function headers?

I felt that keeping that anonymous was an easy way of seeing potential changes we should make in the interface in the future.

Ditto for all the other occurrences of this too.